files/goquorum/config/goquorum/entrypoint.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GOQUORUM_CONS_ALGO=`echo "${GOQUORUM_CONS_ALGO}" | tr '[:lower:]'`
+
+if [ "istanbul" == "$GOQUORUM_CONS_ALGO" ];
```

Comment:
@antonydenyer can we make most of this part of the official quorum image? If we have a bunch of args safe defaulted (say 7-33 eve) that would make management way simpler